web/policies/stripe_account_policy.ex

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.StripeAccountPolicy do
+  import CodeCorps.Helpers.Policy,
+    only: [get_membership: 2, get_role: 1, admin_or_higher?: 1, owner?: 1]
+
+  alias CodeCorps.StripeAccount
+  alias CodeCorps.User
+  alias Ecto.Changeset
+
+  def index?(%User{admin: true}, %StripeAccount{}), do: true
+  def index?(%User{} = user, %StripeAccount{} = stripe_account), do: stripe_account |> get_membership(user) |> get_role |> admin_or_higher?
```

Comment:
For here, I'm not sure yet until we decide what we're doing on `filter` and I get some weigh-in with @begedin. This is likely going to be more complex than that.